otaserver/app.py

```diff
@@ -40,6 +42,10 @@ def run(arguments=[]):
 
     parse_command_line()
 
+    # Custom port for CoAP overrides the one for CoAPS IF set
```

Comment:
This comment is hard to understand. I would rephrase it like this:
Use CoAPs default port except if a custom CoAP port is set.